src/Microsoft.Extensions.PlatformAbstractions/RuntimeEnvironmentExtensions.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Microsoft.Extensions.PlatformAbstractions
+{
+    public static class RuntimeEnvironmentExtensions
+    {
+        public static string GetRuntimeIdentifier(this IRuntimeEnvironment self)
```

Comment:
Good catch. Yes, we'll support both (for now at least) `DNX_RUNTIME_ID` and `DOTNET_RUNTIME_ID` to allow overrides.